api-docs/service-management-intro.md

```diff
@@ -73,7 +73,7 @@ Try it out using the following basic steps
 
 Contact us to acquire a license for the Service Management capability. Your service network will be created during enablement. You can decide which of your accounts should be part of your network. The service network owner must be hierarchically above your service providing accounts as shown in the figure below. After enablement all service providers will have access to the Service Management app within Manager.
```

Comment:
We could consider updating this info with something like: 
"After acquiring the Service Mgmt capability, start off by creating a Service Provider Network."

Reply:
Good point that they are able to do this by themself now.